gctscript/examples/ta/mfi.gct

```diff
@@ -0,0 +1,15 @@
+fmt := import("fmt")
+exch := import("exchange")
+t := import("times")
+mfi := import("indicator/mfi")
+
+load := func() {
+    start := t.date(2017, 8 , 16 , 0 , 0 , 0, 0)
```

Comment:
maybe 16 ->17

Reply:
yep will make this change now missed this one thanks!